nest/iface.c

```diff
@@ -301,6 +301,13 @@ if_update(struct iface *new)
   WALK_LIST(i, iface_list)
     if (!strcmp(new->name, i->name))
       {
+    /* if interface need to be deleted, delete from list and return */
```

Comment:
Indentation looks a bit off here.

Reply:
The indentation is pretty screwed up in this file as-is